sputnikdao2/src/bounties.rs

```diff
@@ -82,7 +85,7 @@ impl Contract {
                 format!("Bounty {} payout", id),
                 None,
             );
-            if bounty.times == 0 {
+            if bounty.times <= 1 {
```

Comment:
I believe this change allows the bounty "done" proposal to be cleared up, and thus that proposal's bond to be returned